### DISCUSSION TITLE
Image Embeddings using base64 as input instead of path URL?

### DISCUSSION THREAD

**Discussion by Matheus-Garbelini** [michaelfeil/infinity#387] [Q&A] [answered]
Hi, I was reading the documentation and checking the API specifications but I'm not sure if it possible to pass image to the clip embeddings via base64 string. Do we always need to pass images via its URL?

**Comment by michaelfeil** [comment, marked as answer]
There is a current PR #385 for this ongoing

**Reply by Matheus-Garbelini** [reply]
Just saw that release [0.0.59](https://github.com/michaelfeil/infinity/releases/tag/0.0.59) includes this features.
Awesome.

**Reply by michaelfeil** [reply]
@Matheus-Garbelini With #395 you might be able to send all of your requests `/embeddings` endpoint. 

Looking for people to try it out before I release it as 0.0.60 version (in case its a release, I would like to keep backwards compatability)
